js/ui/date_box/ui.date_box.base.js

```diff
@@ -510,10 +509,8 @@ const DateBox = DropDownEditor.inherit({
     },
 
     _validateValue: function(value) {
```

Comment:
The `_validateValue` method is called from one place now. Let's remove this method.

Reply:
fixed